docs/releasing.md

```diff
@@ -115,11 +115,11 @@ include/k4arecord/record.h                                     | :white_check_ma
 include/k4arecord/types.h                                      | :white_check_mark: | :white_check_mark: |                    | :white_check_mark: |
 linux-ubuntu/x64/release/libdepthengine.so \*                  |                    |                    | :white_check_mark: |                    |
 linux-ubuntu/x64/release/libdepthengine.so.1.0 \*              |                    |                    | :white_check_mark: |                    |
-linux-ubuntu/x64/release/libk4a.so                             |                    |                    | :white_check_mark: |                    |
-linux-ubuntu/x64/release/libk4a.so.1.1                         |                    |                    | :white_check_mark: |                    |
+linux-ubuntu/x64/release/libk4a.so (symlink)                   |                    |                    |                    | :white_check_mark: |
+linux-ubuntu/x64/release/libk4a.so.1.1 (symlink)               |                    |                    |                    | :white_check_mark: |
```

Comment:
The symlink for libk4a.so.1.1 -> libk4a.so.1.1.0 still exists in the runtime package. It must because our dependencies will look for libk4a.so.1.1 (not libk4a.so.1.1.0).

```suggestion
linux-ubuntu/x64/release/libk4a.so.1.1 (symlink)               |                    |                    | :white_check_mark: |                   |
```